aspnetcore/security/authentication/index.md

```diff
@@ -148,6 +148,15 @@ See the [Orchard Core](https://github.com/OrchardCMS/OrchardCore) source for an
 
 [ABP Framework](https://abp.io/) supports various architectural patterns including modularity, microservices, domain driven design, and multi-tenancy. See [ABP Framework source on GitHub](https://github.com/abpframework/abp).
 
+Finbuckle.MultiTenant:
+
+* Open source
+* Provides tenant resolution
+* Lightweight
+* Provides data isolation
+* Configure app behavior uniquely for each tenant
```

Comment:
@AndrewTriesToCode  I just took those from the web page. You might want to have a list better differentials Finbuckle.MultiTenant from the previous alternatives.